types/RootEngine.d.ts

```diff
@@ -0,0 +1,16 @@
+export default RootEngine;
+declare class RootEngine {
+    constructor(config: any, resolveInputControls: any, fireNodeFunction: any);
+    config: any;
+    fireNodeFunction: any;
+    resolveInputControls: any;
```

Comment:
It would also be nice to type fireNodeFunction and resolveInputControls, because they are supposed to follow a certain schema. We might be able to create a union type from the types specified when the nodes are created to make the user handle all the cases. I might try this out later. This is not a priority for basic types.